boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853_defconfig

```diff
@@ -15,13 +15,6 @@ CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_PINMUX=y
 CONFIG_GPIO=y
-CONFIG_PS2=y
-CONFIG_ADC=y
-CONFIG_PWM=y
-CONFIG_SENSOR=y
-CONFIG_I2C=y
-CONFIG_I2C_INIT_PRIORITY=60
```

Comment:
I suggest putting removal of CONFIG_I2C_INIT_PRIORITY in a different PR.

Reply:
added specific commit, please check